lib/hooks/bitbucket/templates/issue_comment_created.html.haml

```diff
@@ -0,0 +1,16 @@
+:ruby
+  issue   = payload.issue
+  actor   = payload.actor
+  comment = payload.comment
+
+%p
+  %span
+    = avatar_image_tag(actor.links.avatar.href)
+
+  %a{href: actor.links.html.href}
+    = actor.display_name
+  created issue comment to
+  %a{href: comment.links.html.href}
+    #{comment.id}
```

Comment:
How about using `#{payload.repository.full_name}##{issue.id}`?

Like this:
![selection_354](https://cloud.githubusercontent.com/assets/43346/11261731/a03a2c46-8eba-11e5-8309-b227249915ad.png)